src/cgroup.c

```diff
@@ -306,33 +306,39 @@ gboolean check_cgroup2_oom()
 			continue;
 
 		if (counter != last_counter) {
-			if (write_oom_files() == 0)
+			if (create_oom_files() == 0)
 				last_counter = counter;
 		}
 		return G_SOURCE_CONTINUE;
 	}
 	return G_SOURCE_REMOVE;
 }
 
-/* write the appropriate files to tell the caller there was an oom event
- * this can be used for v1 and v2 OOMS
+static int create_oom_file(const char *base_path)
```

Comment:
nit: can this be defined underneath where it's called?

Reply:
done